inference-engine/src/mkldnn_plugin/utils/README.md

```diff
@@ -35,8 +37,22 @@ Options are:
 * BIN (default)
 * TEXT
 
+### Filter input / output blobs
+To dump only input / output blobs:
+```sh
+    OV_CPU_BLOB_DUMP_NODE_PORTS='<ports_kind>' binary ...
+```
+Example:
+```sh
+    OV_CPU_BLOB_DUMP_NODE_PORTS=OUT binary ...
+```
+Options are:
```

Comment:
Looking forward - if we will allow to conditionally dump some internal data (like internal blobs), isn't it better to support enumeration (like ***=IN OUT INTERNAL).

Reply:
I have started with ENUM implementation but it makes code less clean.
I mean we have dumpFilters as map<int, strings>, it is easier to keep input/output filter in strings as well.
If we introduce new container for input/output filter, then the first check in "shouldBeDumped" function will be much more complex.
Basically this comment can be applicable for all the filters. We could use enum for filter by types, int->int for filter by exec id, etc.
My proposal to keep it as it is.

Reply:
Discussed offline: changing `BOTH` -> `ALL`. Rest keep as is.

Reply:
Done